extensions/grpc/deployment/pom.xml

```diff
@@ -39,6 +43,98 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-health-spi</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
```

Comment:
Wouldn't it download all the variants even if you need only the one for the current OS/Arch?
Maybe we can resolve it programmatically.

Reply:
I started by doing it dynamically but was advised otherwise for simplicity.
AFAIR I had problems making dynamic resolution work with Gradle.